### PR TITLE
Correctly map main entry literals.

### LINKED ISSUE #170: MARC conversion--Not adding author with literal and no URI
The MARC conversion is not creating a 1XX field for the primary contributor when the primary contributor is only entered as a literal with no corresponding URI. For example, see the following work resource: https://api.stage.sinopia.io/resource/f0f8a70a-74f5-44f2-9096-c50c81233e7b. The author is there as a literal, but does not end up in the converted MARC record. I'm attaching screen shots of the work instance with the author, the section of the RDF preview with the author's name in the RDF, and of the converted MARC record.
Creator as literal
RDF with author as literal
Record with missing 100

### PR DESCRIPTION
**Pull request by justinlittman**
closes #170

## Why was this change made?
Correct mapping.

### CODE REVIEW

File: spec/rdf2marc/rdf2model/mappers/main_entry_fields_spec.rb [hunk 1]
```diff
@@ -77,7 +77,7 @@
           _:b7 a <http://id.loc.gov/ontologies/bflc/PrimaryContribution>;
               <http://id.loc.gov/ontologies/bibframe/agent> _:b8.
           _:b8 a <http://id.loc.gov/ontologies/bibframe/Person>;
-              <http://www.w3.org/1999/02/22-rdf-syntax-ns#value> "Jung, Carl", "Kennedy Family".
+              <http://www.w3.org/2000/01/rdf-schema#label> "Jung, Carl", "Kennedy Family".
```
Comment (peetucket) [inline spec/rdf2marc/rdf2model/mappers/main_entry_fields_spec.rb:80]: is this a separate unrelated fix?

**Review by ndushay — APPROVED** [review]

File: spec/rdf2marc/rdf2model/mappers/main_entry_fields_spec.rb (hunk 1 above)
Comment (justinlittman) [reply spec/rdf2marc/rdf2model/mappers/main_entry_fields_spec.rb:80]: No, that fixes the test so that it tests the correct thing.
